app/src/main/java/me/quadphase/qpdex/databaseAccess/PokemonFactory.java

```diff
@@ -0,0 +1,248 @@
+package me.quadphase.qpdex.databaseAccess;
```

Comment:
Note to possible other reviewers: A lot of this class will be changed when I push my stuff from Issue #12